_includes/_components/doctrine-events.html

```diff
@@ -2,10 +2,11 @@
     <header>
         <div class="container">
             <h2>Doctrine Events</h2>
-            <p>Access the easy way to your doctrine events from the Symfony DIC.</p>
+            <p>A single events doctrine dispatcher in the Symfony dispatcher</p>
```

Comment:
`A single doctrine events dispatcher in the Symfony one.` ?
`Doctrine to Symfony events redispatcher`?